src/SFA.DAS.Commitments.Database/StoredProcedures/SetTransferApproval.sql

```diff
@@ -1,37 +0,0 @@
-﻿CREATE PROCEDURE [dbo].[SetTransferApproval]
```

Comment:
I was wondering what could happen by removing this sproc during a zero downtime deployment, but we update the database *after* the cloud deployment, so this should be fine.